src/styles/20-tools/_mixins/_pseudo.scss

```diff
@@ -68,3 +68,30 @@
     @content;
   }
 }
+/**
+ * Fallback for :focus
+ *
+ * Usage:
+ *
+ * .selector {
+ *   @include outline;
+ * }
+ *
+ */
+
+@mixin outline {
+  &:focus {
```

Comment:
@RhodesPeter @creido-welly, I have updated my code as with the previous version the focus ring was shown when a mouse was used.

> This code snippet can prevent from unexpected browser's behaviour, when it can apply :**focus** without **:focus-visible**.

Source: [Is it time for :focus-visible?](https://www.dnbtech.no/2023/is-it-time-for--focus-visible-)

I tested [this codepen](https://codepen.io/skibinska/pen/WNYLmpY) in Chrome, Firefox and Safari and it is working as expected.